tests/models/test_utils.py

```diff
@@ -260,14 +260,12 @@ def test_enforce_property():
     prop = Property("a", Array(DataType.string))
     assert _enforce_property(data, prop) == data
 
-    data = np.array(["some_sentence1", "some_sentence2"])
-    assert _enforce_property(data, Property("a", Array(DataType.string))) == [
-        "some_sentence1",
-        "some_sentence2",
-    ]
-
     prop = Property("a", Array(DataType.binary))
-    assert _enforce_property(data, prop) == ["some_sentence1", "some_sentence2"]
+    assert _enforce_property(data, prop) == data
+
+    data = np.array([np.int32(1), np.int32(2)])
+    prop = Property("a", Array(DataType.integer))
+    assert (_enforce_property(data, prop) == data).all()
```

Comment:
/ai give me pytest code to compare `assert (a == b).all()` and `np.testing.assert_equal(a, b)`

Reply:
`(a == c).all()` produces a better error message:


```
============================================================================================= FAILURES ==============================================================================================
_________________________________________________________________________________________ test_assertions_1 _________________________________________________________________________________________

    def test_assertions_1():
        a = np.array([1, 2, 3, 4, 5])
        c = np.array([1, 2, 3, 4, 6])
    
>       assert (a == c).all()  # This will fail since a and c are not equal
E       assert False
E        +  where False = <built-in method all of numpy.ndarray object at 0x7f06a6e58870>()
E        +    where <built-in method all of numpy.ndarray object at 0x7f06a6e58870> = array([1, 2, 3, 4, 5]) == array([1, 2, 3, 4, 6])
E             Full diff:
E             - array([1, 2, 3, 4, 6])
E             ?                    ^
E             + array([1, 2, 3, 4, 5])
E             ?                    ^.all

a          = array([1, 2, 3, 4, 5])
c          = array([1, 2, 3, 4, 6])

a.py:8: AssertionError
_________________________________________________________________________________________ test_assertions_2 _________________________________________________________________________________________

    def test_assertions_2():
        a = np.array([1, 2, 3, 4, 5])
        c = np.array([1, 2, 3, 4, 6])
    
>       np.testing.assert_equal(a, c)  # This will fail since a and c are not equal
E       AssertionError: 
E       Arrays are not equal
E       
E       Mismatched elements: 1 / 5 (20%)
E       Max absolute difference: 1
E       Max relative difference: 0.16666667
E        x: array([1, 2, 3, 4, 5])
E        y: array([1, 2, 3, 4, 6])

a          = array([1, 2, 3, 4, 5])
c          = array([1, 2, 3, 4, 6])
```